src/table/body-cell/disabled-inline-editor.tsx

```diff
@@ -72,13 +69,8 @@ export function DisabledInlineEditor<ItemType>({
       nativeAttributes={
         { 'data-inline-editing-active': isEditing.toString() } as TableTdElementProps['nativeAttributes']
       }
-      className={clsx(
-        className,
-        styles['body-cell-editable'],
-        resizableColumns && styles['resizable-columns'],
-        isEditing && styles['body-cell-edit-disabled-popover'],
```

Comment:
This style is now added in the TableTdElement: `isEditing && isEditingDisabled && styles['body-cell-edit-disabled-popover']`